src/generator.ts

```diff
@@ -47,10 +47,7 @@ export async function generateServer(
     )
     .join(',\n');
 
-  const serverCode = `
-#!/usr/bin/env node
```

Comment:
@openhands-agent the shebang shouldn't have been removed. The generated server needs to be ran via command line.